src/locale/fr_FR.ts

```diff
@@ -241,39 +291,42 @@ export default {
       X: 'X'
     }
   },
-  GsColorMapEntryField: {
+  ColorMapEntryField: {
     colorLabel: 'Couleur',
     labelLabel: 'Étiquette de légende',
     quantityLabel: 'Quantité',
     opacityLabel: 'Opacité'
   },
-  GsChannelField: {
+  ChannelField: {
     sourceChannelNameLabel: 'Nom du canal',
     contrastEnhancementTypeLabel: 'Amélioration du contraste',
     gammaValueLabel: 'Transparence (Gamma)'
   },
-  GsColorMapTypeField: {
+  ColorMapTypeField: {
     rampMapTypeLabel: 'Interpolé',
     intervalsMapTypeLabel: 'Intervalles',
     valuesMapTypeLabel: 'Valeurs'
   },
-  GsSymbolizerEditor: {
+  SymbolizerEditor: {
     kindFieldLabel: 'Type'
   },
-  GsSymbolizerEditorWindow: {
+  SymbolizerEditorWindow: {
     symbolizersEditor: 'Éditeur de symbolisation'
   },
-  GsFilterEditorWindow: {
+  FilterOverview: {
+    filterTitle: 'Filters'
```

Comment:
Is this really French?

Reply:
I guess sometimes its more natural to keep the english word. But i guess we're the wrong persons to discuss this. :rofl:

Reply:
So this is a translation that was already provided?